pkg/proxy/nftables/proxier.go

```diff
@@ -1571,20 +1580,6 @@ func (proxier *Proxier) syncProxyRules() {
 		klog.ErrorS(err, "Failed to list nftables chains: stale chains will not be deleted")
 	}
 
-	// OTOH, we can immediately delete any stale affinity sets
```

Comment:
You will need to replace the stale-affinity-set deletion with stale-affinity-map deletion.

Reply:
Acked. The new per-service affinity-map can be stale in the scenario when the service with configuration changes is re-loaded. This is a bit different with the case that the origin per-endpoint affinity-set is stale when only endpoints of the service are changed.

I'll try to handle the stale-affinity-map deletion in the next version.

Reply:
You're right. I've added back the deletion codes for the stale-affinity map to the new commit version.